curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/understand-the-prototype-chain.english.md

```diff
@@ -14,21 +14,20 @@ function Bird(name) {
   this.name = name;
 }
 
-typeof Bird.prototype; // => object
+typeof Bird.prototype; // yields 'object'
 ```
 
 Because a <code>prototype</code> is an object, a <code>prototype</code> can have its own <code>prototype</code>! In this case, the <code>prototype</code> of <code>Bird.prototype</code> is <code>Object.prototype</code>:
 
 ```js
-Object.prototype.isPrototypeOf(Bird.prototype);
-// returns true
+Object.prototype.isPrototypeOf(Bird.prototype); // returns true
```

Comment:
```suggestion
Object.prototype.isPrototypeOf(Bird.prototype); // yields true
```
It would be good to use one notation throughout, other snippets use 'yields'. Changing others to use 'returns' is also an option.

Reply:
This is/was handled in one of the other PRs.  I used yields across the board when relevant.